proto/ibc/core/client/v1/query.proto

```diff
@@ -255,6 +255,6 @@ message QueryClientRequest {
 
 // QueryClientRequest is the response type for the Query/Client RPC method
 message QueryClientResponse {
-  string       creator      = 1;
-  Counterparty counterparty = 2 [(gogoproto.nullable) = false];
+  string creator = 1;
+  // ibc.core.packetserver.v1.Counterparty counterparty = 2 [(gogoproto.nullable) = false];
```

Comment:
Commented this out to prevent the circular dependency I mentioned above. We might need to move `Counterparty` to a different place.